src/themes/fa.css

```
@@ -459,47 +568,73 @@ wa-button:not([variant='text']) {
   }
 }
 
-/* not sure if i'm tying into something stable here, or if this is the right way to do this */
-wa-checkbox[checked] {
-  --wa-color-brand-text-on-spot: white;
-}
-wa-radio::part(control--checked) {
-  --wa-color-brand-text-on-spot: white;
-}
+wa-card {
+  --border-width: 0px;
 
-wa-radio-group > wa-radio-button,
-wa-radio-group > wa-button {
-  --box-shadow: none;
+  & .title {
+    font-size: var(--wa-font-size-xl);
+  }
+
+  & wa-rating {
+    --symbol-size: var(--wa-font-size-m);
+  }
 }
 
-wa-switch {
-  --background: var(--wa-color-neutral-spot);
-  --border-color: var(--wa-color-neutral-spot);
+wa-carousel {
+  --pagination-color-activated: var(--wa-color-brand-spot);
```

Unrequested opinion... what if the resting color was a little lighter?
```suggestion
  --pagination-color-activated: var(--wa-color-brand-spot);
  --pagination-color-resting: var(--wa-color-gravy-80);  
```